include/swift/AST/DiagnosticsSema.def

```diff
@@ -436,6 +436,9 @@ ERROR(unspaced_unary_operator,sema_nb,none,
 
 ERROR(use_unresolved_identifier,sema_nb,none,
       "use of unresolved %select{identifier|operator}1 %0", (Identifier, bool))
+WARNING(confusable_character,sema_nb,none,
```

Comment:
Rather than include this in two different Diagnostics files, please move it to DiagnosticsCommon.def.